setup.cfg

```diff
@@ -93,6 +93,9 @@ all =
     ipython>=4.2
     pytest>=7.0
     typing_extensions>=3.10.0.1
+    fsspec>=2022.1.0,!=2022.7.1
+    aiohttp!=4.0.0a0,!=4.0.0a1  # dependency of fsspec with broken alpha releases
```

Comment:
Is there a way to specify fsspec version so that we don't need aiohttp pin here?

Reply:
Yes, I fixed this upstream via https://github.com/fsspec/filesystem_spec/pull/1018, so we can replace both lines with `fsspec[http]>new_version` as soon as a new version of fsspec is released on pypi.  We can likely ping @martindurant for that.